.github/workflows/gh-ci-cron.yaml

```diff
@@ -196,3 +196,71 @@ jobs:
     - name: run_tests
       run: |
         pytest -n $numprocs testsuite/MDAnalysisTests --disable-pytest-warnings --durations=50
+        
+  # Issue 4208
+  conda-latest-release:
+    # A set of runner to check that the latest conda release works as expected
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+        fail-fast: false
+        matrix:
+          os: [ubuntu-latest, macos-latest]
+          python-version: ["3.9", "3.10", "3.11"]
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
Do we need the checkout action? We don’t use the repo, do we?

Reply:
Yeah we need it to pick up our custom actions (setup-deps in this case).

Reply:
ok!